package.json

```diff
@@ -61,6 +62,7 @@
     "@types/react": "16.9.25",
     "@types/react-dom": "16.9.5",
     "@types/styled-components": "4.4.3",
+    "@types/util-deprecate": "^1.0.0",
```

Comment:
I think if `util-deprecate` it's imported to the production bundle that should go deps not devDeps